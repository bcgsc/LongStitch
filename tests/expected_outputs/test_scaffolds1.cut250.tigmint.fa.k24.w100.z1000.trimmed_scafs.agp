ntLink_0	1	8001	1	W	180793-1::180793:0-8001	1	8001	-
ntLink_0	8002	8021	2	N	20	scaffold	yes	paired-ends
ntLink_0	8022	11993	3	W	187066::187066:0-3972	1	3972	-
ntLink_0	11994	27763	4	N	15770	scaffold	yes	paired-ends
ntLink_0	27764	66772	5	W	189924::189924:0-39009	1	39009	+
ntLink_0	66773	67311	6	N	539	scaffold	yes	paired-ends
ntLink_0	67312	68528	7	W	187638::187638:0-1217	1	1217	+
ntLink_0	68529	69631	8	N	1103	scaffold	yes	paired-ends
ntLink_0	69632	188236	9	W	189963::189963:0-118605	1	118605	+
ntLink_0	188237	188256	10	N	20	scaffold	yes	paired-ends
ntLink_0	188257	208113	11	W	181193::181193:0-23261	3405	23261	-
ntLink_0	208114	257217	12	W	190623::190623:0-62735	13632	62735	+
ntLink_0	257218	271581	13	N	14364	scaffold	yes	paired-ends
ntLink_0	271582	293767	14	W	187190::187190:0-22186	1	22186	+
ntLink_0	293768	307378	15	N	13611	scaffold	yes	paired-ends
ntLink_0	307379	321352	16	W	182499::182499:0-14036	63	14036	-
ntLink_0	321353	324762	17	W	185554::185554:0-3448	39	3448	+
ntLink_0	324763	325517	18	N	755	scaffold	yes	paired-ends
ntLink_0	325518	326646	19	W	103642::103642:0-1129	1	1129	+
ntLink_0	326647	326670	20	N	24	scaffold	yes	paired-ends
ntLink_0	326671	330370	21	W	187264::187264:0-3700	1	3700	-
ntLink_1	1	22391	1	W	183815::183815:0-22391	1	22391	+
ntLink_1	22392	23825	2	N	1434	scaffold	yes	paired-ends
ntLink_1	23826	80711	3	W	190275::190275:0-56919	34	56919	-
ntLink_1	80712	163592	4	W	185693::185693:0-82911	31	82911	+
185898::185898:0-6886	1	6886	1	W	185898::185898:0-6886	1	6886	+
180793-2::180793:8001-8024	1	23	1	W	180793-2::180793:8001-8024	1	23	+
180793-3::180793:8024-12931	1	4907	1	W	180793-3::180793:8024-12931	1	4907	+
182485::182485:0-7490	1	7490	1	W	182485::182485:0-7490	1	7490	+
