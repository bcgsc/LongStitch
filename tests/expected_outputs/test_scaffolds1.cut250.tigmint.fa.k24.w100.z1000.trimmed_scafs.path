ntLink_0	180793-1::180793:0-8001- 21N 187066::187066:0-3972- 15764N 189924::189924:0-39009+ 547N 187638::187638:0-1217+ 1094N 189963::189963:0-118605+ 21N 181193::181193:0-23261- 21N 190623::190623:0-62735+ 14387N 187190::187190:0-22186+ 21N 182485::182485:0-7490- 458N 185898::185898:0-6886- 160N 182499::182499:0-14036- 21N 185554::185554:0-3448+ 758N 103642::103642:0-1129+ 23N 187264::187264:0-3700- 22459N 183815::183815:0-22391+ 1439N 190275::190275:0-56919- 21N 185693::185693:0-82911+
