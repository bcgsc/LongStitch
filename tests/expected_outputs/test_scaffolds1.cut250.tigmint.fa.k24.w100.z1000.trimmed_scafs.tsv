103642::103642:0-1129	0	1129
185898::185898:0-6886	0	6886
190623::190623:0-62735	13631	62735
180793-1::180793:0-8001	0	8001
180793-2::180793:8001-8024	0	23
180793-3::180793:8024-12931	0	4907
187066::187066:0-3972	0	3972
181193::181193:0-23261	3404	23261
187190::187190:0-22186	0	22186
182485::182485:0-7490	0	7490
187264::187264:0-3700	0	3700
182499::182499:0-14036	62	14036
187638::187638:0-1217	0	1217
183815::183815:0-22391	0	22391
189924::189924:0-39009	0	39009
185554::185554:0-3448	38	3448
189963::189963:0-118605	0	118605
185693::185693:0-82911	30	82911
190275::190275:0-56919	33	56919
